src/pvr2wmc.h

```diff
@@ -1,25 +1,23 @@
-#pragma once
 /*
-*      Copyright (C) 2011 Pulse-Eight
-*      http://www.pulse-eight.com/
```

Comment:
I guess it was c+p'd from the pvr.demo add-on in that case, which I originally wrote as a demo for Pulse-Eight. This header doesn't contain any of that code any more, the others may. Will have to check that. If you don't want to wait for that, then you can just keep both the team and Pulse-Eight in the copyright ("portions copyright Pulse-Eight Limited")